go.mod

```diff
@@ -120,6 +120,8 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
+require k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
```

Comment:
I think you need a `go mod tidy`, otherwise I'm confused why this change caused this 🤔

Reply:
I thought I ran it 🤔 going to do it again